src/logging/watch-logger.ts

```diff
@@ -46,6 +48,10 @@ export class WatchLogger implements Logger {
     }
   }
 
+  printMetrics(): void {
```

Comment:
When this method is empty, eslint complains:

> Unexpected empty method 'printMetrics'.eslint@typescript-eslint/no-empty-function

This could also be resolved by adding `//eslint-disable-line` or updating the ESLint config, but adding a comment seemed pragmatic.